frontend/src/app/main/ui/dashboard/sidebar.scss

```diff
@@ -24,7 +24,7 @@
   margin: 0 var(--sp-l) 0 0;
   border-right: $b-1 solid var(--panel-border-color);
   background-color: var(--panel-background-color);
-  z-index: deprecated.$z-index-1;
+  z-index: deprecated.$z-index-2;
```

Comment:
They're deprecated. We have z-indexes vars in `ds/z-index.css`. Maybe we should take the opportunity to migrate this component to those values? (if it's too much of a hassle, we can leave that for the future)